packages/devtools_app/lib/src/shared/survey.dart

```diff
@@ -58,23 +58,23 @@ class SurveyService {
         NotificationAction(
           _noThanksLabel,
           () => _noThanksPressed(
-            message: message,
+            message: message!,
```

Comment:
unwrap the `message` var where it is defined, and then you won't have to force unwrap it for every use

Reply:
done